tasks/debian.yml

```diff
@@ -0,0 +1,40 @@
+- name: "Install exim (exim4) the mail transfer agent"
+  apt: >
+    name=exim4-daemon-heavy
+    state=present
+  tags:
+    - exim
+
+- name: "Make sure exim (exim4) is running"
+  service: >
+    name=exim4
+    state=started
+    enabled=yes
+  tags:
+    - exim
+
+
+- name: "Configure the package"
+  shell: |
+    executable=/bin/bash
+    dpkg-reconfigure -f noninteractive exim4-config &&\
```

Comment:
Good question... The doc advises to run `dpkg-reconfigure` in any case, but I can try without and see if it works. Hold on.